ui/app/AppLayouts/CommunitiesPortal/popups/DiscordImportProgressContents.qml

```diff
@@ -259,11 +259,18 @@ StatusScrollView {
             Layout.fillWidth: true
             spacing: 12
 
-            Image {
+            StatusLoadingIndicator {
+                Layout.preferredHeight: 24
+                Layout.preferredWidth: 24
+                Layout.alignment: Qt.AlignHCenter
+                visible: root.store.discordImportHasCommunityImage && root.store.discordImportCommunityImage.toString() == ""
+            }
```

Comment:
@caybro added a loading indicator here in case we know there's gonna be an image but the progress signal hasn't emitted an image yet

Reply:
Perfect!